### PR TITLE
fix/158 Handling all potential image types in CSV download

### LINKED ISSUE #158: Query results, download CSV fails if anything but multiple images present in results
Needs to handle none, single as well as multiple in images column.

### PR DESCRIPTION
**Pull request by Robbie1977**
Resolves #158

- [TBD in geppetto-vfb] Add coverage for whatever new functionality, to a JUnit test if it's backend, to a Casper Test if it's frontend
- [x] Make sure both push and pr travis builds are passing before asking for a review

### CODE REVIEW

**Comment by Robbie1977** [comment]
@ddelpiano @tarelli This is ready to go now.

**Comment by Robbie1977** [comment]
Example http://v2.virtualflybrain.org/org.geppetto.frontend/geppetto?id=FBbt_00003748&i=VFB_00017894 
       Transgenes expressed in the medulla
       Neurons with some part in medulla


With prosed fix: http://v2-dev2.virtualflybrain.org/org.geppetto.frontend/geppetto?id=FBbt_00003748&i=VFB_00017894

File: js/components/interface/query/queryBuilder.js [hunk 1]
```diff
@@ -1047,8 +1047,15 @@ define(function (require) {
               continue;
             }
             if (args.columnsPresent[counter] == "images") {
-              instance[args.columnsPresent[counter]] = JSON.parse(record[args.columnsPresent[counter]]).initialValues[0].value.elements[0].initialValue.data;
-              continue;
+              if (record[args.columnsPresent[counter]].indexOf('{') > -1){
```
Comment (ddelpiano) [inline js/components/interface/query/queryBuilder.js]: @Robbie1977 I would change this check inside the if with a function to check if the data can be parsed to be sure the error is handled anyway with a function similar to the one I am putting here below, and use this function to check inside the if.
The rest is good for me.
Thx! 
```
var isJson = data => {
          try {
            return (JSON.parse(data) && data);
          } catch (e) {
            return false;
          }
        }
```

**Comment by Robbie1977** [comment]
@ddelpiano @tarelli this one also looks good to be merged?

**Pull request merged**